src/preferences/dialog/dlgprefbroadcastdlg.ui

```diff
@@ -333,187 +312,257 @@ p, li { white-space: pre-wrap; }
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+          </property>
+         </widget>
+        </item>
+        <item row="4" column="0">
+         <widget class="QLabel" name="label_13">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
           <property name="text">
-           <string notr="true">%maintitle</string>
+           <string>Genre</string>
+          </property>
+          <property name="buddy">
+           <cstring>stream_genre</cstring>
           </property>
          </widget>
         </item>
-       </layout>
-      </item>
-     </layout>
-    </widget>
-   </item>
-   <item row="1" column="0" colspan="2">
-    <widget class="QGroupBox" name="groupBox">
-     <property name="title">
-      <string>Server connection</string>
-     </property>
-     <layout class="QGridLayout">
-      <item row="1" column="0">
-       <layout class="QGridLayout" name="gridLayoutSC">
-        <item row="0" column="1">
-         <widget class="QComboBox" name="comboBoxServerType">
+        <item row="4" column="1">
+         <widget class="QLineEdit" name="stream_genre">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
+          <property name="text">
+           <string>Live mix</string>
+          </property>
          </widget>
         </item>
-        <item row="0" column="0">
-         <widget class="QLabel" name="label">
+        <item row="0" column="0" colspan="2">
+         <widget class="QCheckBox" name="stream_public">
           <property name="text">
-           <string>Type</string>
+           <string>Public stream</string>
           </property>
          </widget>
         </item>
-        <item row="1" column="0">
-         <widget class="QLabel" name="label_2">
+       </layout>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item row="3" column="1">
+    <widget class="QGroupBox" name="groupBoxMetadata">
+     <property name="title">
+      <string>Metadata</string>
+     </property>
+     <property name="flat">
+      <bool>false</bool>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout">
+      <item>
+       <layout class="QGridLayout" name="gridLayout">
+        <item row="0" column="0">
+         <widget class="QLabel" name="label_16">
           <property name="text">
-           <string>Host</string>
+           <string>Format  </string>
```

Comment:
The new ``retries`` block should align with the existing``coding`` block in the grid layout. Currently the ``coding`` block is misaligned.